backend/controllers/retentionReasons.js

```diff
@@ -1,24 +1,27 @@
+/* eslint-disable consistent-return */
```

Comment:
A few helper methods here can return null

Reply:
better to not blanket ignore the whole file as the warnings. These are generally easy to fix, which places in particular?

Reply:
I'll have another look, the linter wasn't having various different arrangements of one function.